chain/client/src/metrics.rs

```diff
@@ -127,7 +127,7 @@ pub(crate) static VALIDATORS_CHUNKS_EXPECTED_IN_EPOCH: Lazy<IntGaugeVec> = Lazy:
     try_create_int_gauge_vec(
         "near_validators_chunks_expected_in_epoch",
         "Number of chunks expected to be produced by a validator within current epoch",
-        &["account_id", "shard_id"],
+        &["account_id", "shard_id", "epoch_id"],
```

Comment:
Let's use `epoch_height` here and below because there is an assumption in the code that `epoch_id` is the _hash_ defining the epoch.

Reply:
good catch! I must have missed it when I rebased my changes to main branch.